app/views/content/landing/career-progression/_collage.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-full-content">
+      <div class="inset">
+        <p class="col-720">
+Teaching is fulfilling career. Not only can you help the pupils you teach, you also get to learn new skills, build your confidence and become your best self. The benefits of being a teacher include:         </p>
+
+        <%= render Content::ChecklistCollageComponent.new(
+          checklist: [
+            "career progression ",
+            "personal development",
+            "teacher's pay"
```

Comment:
```suggestion
            "£30k minimum starting salary"
```
this feels a bit woolly at the moment?

I appreciate it is spelled out further down the page and that you are linking bullets to subsequent h2s so my suggestion might be too specific! If so, I think we need to strengthen this to be more about opportunities to progess pay etc.

Reply:
@gemmadallmandfe I've amended to: Opportunities to increase your pay and also amended the section header so it matches.